scripts/generate-patch

```diff
@@ -34,8 +34,8 @@ function revert_crd_changes {
 
 
 for f in packages/*; do
-  if [[ -f ${f}/package.yaml ]]; then
-  	if [[ -z $CHART || $CHART == $(basename -- ${f}) ]]; then
+	if [[ (-f ${f}/package.yaml) && ($(cat ${f}/package.yaml | yq r - url)) ]]; then
+		if [[ -z $CHART || $CHART == $(basename -- ${f}) ]]; then
 			url=$(cat ${f}/package.yaml | yq r - url)
```

Comment:
Instead of checking it in the for loop above, maybe we should just check something like:
```suggestion
	if [[ -f ${f}/package.yaml ]]; then
		if [[ -z $CHART || $CHART == $(basename -- ${f}) ]]; then
			url=$(cat ${f}/package.yaml | yq r - url)
			if [[ -z ${url} ]]; then
			    continue
			fi
```